### DISCUSSION TITLE
Error CS0012: The type 'Color' is defined in an assembly that is not referenced

### DISCUSSION THREAD

**Discussion by FoggyFinder** [AvaloniaUI/Avalonia#5389] [Q&A] [answered]
Got an error [during build](https://github.com/verybadcat/CSharpMath/pull/187) on CI

```
Error: Test.cs(16,9): error CS0012: The type 'Color' is defined in an assembly that is not referenced. You must add a reference to assembly 'Avalonia.Visuals, Version=0.10.0.0, Culture=neutral, PublicKeyToken=null'. [/home/runner/work/CSharpMath/CSharpMath/CSharpMath.Xaml.Tests.NuGet/CSharpMath.Xaml.Tests.NuGet.csproj]
```

I wasn't able to reproduce it locally though. But I tested only on Windows.

Any ideas are welcome I haven't a clue why it happens.

**Comment by kekekeks** [comment, marked as answer]
You have some dll built for 0.9.x branch (it might be a nuget reference). Recompile everything to 0.10.0.

**Reply by FoggyFinder** [reply]
No, but there is a reference to `0.10.0-preview5` version. And since `0.10-rc` version they are not compatible, right? Is so then I suppose it is not a bug but expected behavior.

**Reply by kekekeks** [reply]
Ah, yes, strong naming was first added in rc1
